status.tmpl

```diff
@@ -0,0 +1,97 @@
+<html>
+  <head>
+    <style type="text/css">
```

Comment:
Let's break the css into a separate file, in order to clean this it up a bit. (Makes it easier to focus on just the template.)